srp_daemon/srp_daemon.c

```diff
@@ -1991,12 +2045,16 @@ static int ibsrpdm(int argc, char *argv[])
 		}
 	}
 
+	ret = setup_wakeup_fd();
```

Comment:
I did this only because the original code setup wakeup_fd before calling ibsrpdm, and I saw stuff in ibsrpdm that looks like it was using threads. If you are certain then I will drop it.